### PR TITLE
Add Save Flag to Documentation

### PR DESCRIPTION
**Pull request by ev1stensberg**
Adds a new flag to the documentation and a section about the CLI. Check
if the PR regarding save has been merged yet.

### CODE REVIEW

File: content/api/cli.md [hunk 1]
```diff
@@ -163,6 +167,7 @@ These options makes the build watch for changes in files of the dependency graph
 | Parameter                 | Explanation                                             |
 |---------------------------|---------------------------------------------------------|
 | --watch, -w               | Watch the filesystem for changes                        |
+| --save, -s                | Recompiles on save regardless of changes                |
```
Comment (SpaceK33z) [inline content/api/cli.md]: This flag doesn't exist at all? Could you link me to the code if it does exist?

Comment (ev1stensberg) [inline content/api/cli.md]: - https://github.com/webpack/webpack/pull/3576

**Comment by bebraw** [comment]
Can you rebase against `master`? The development happens against it now (faster deployments).

**Comment by ev1stensberg** [comment]
@bebraw Like that?

**Comment by bebraw** [comment]
The first commit feels redundant. I think you can eliminate it with

```bash
git rebase master
git push --force
```

Force push is fine in this case as it's a feature branch.

**Pull request closed without merging**